src/panels/config/automation/trigger/types/ha-automation-trigger-numeric_state.ts

```diff
@@ -1,22 +1,36 @@
-import "@polymer/paper-input/paper-input";
-import "@polymer/paper-input/paper-textarea";
+import "../../../../../components/ha-form/ha-form";
 import { html, LitElement, PropertyValues } from "lit";
 import { customElement, property } from "lit/decorators";
+import memoizeOne from "memoize-one";
 import { createDurationData } from "../../../../../common/datetime/create_duration_data";
 import { fireEvent } from "../../../../../common/dom/fire_event";
 import { hasTemplate } from "../../../../../common/string/has-template";
-import "../../../../../components/entity/ha-entity-picker";
-import { NumericStateTrigger } from "../../../../../data/automation";
-import { HomeAssistant } from "../../../../../types";
-import { handleChangeEvent } from "../ha-automation-trigger-row";
-import "../../../../../components/ha-duration-input";
+import type { NumericStateTrigger } from "../../../../../data/automation";
+import type { HomeAssistant } from "../../../../../types";
+import type { HaFormSchema } from "../../../../../components/ha-form/types";
 
 @customElement("ha-automation-trigger-numeric_state")
 export class HaNumericStateTrigger extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
   @property() public trigger!: NumericStateTrigger;
 
+  private _schema = memoizeOne((entityId) => [
```

Comment:
Make sure you type this function so we get validation.
```suggestion
  private _schema = memoizeOne((entityId): HaForm[] => [
```